src/components/ButtonComponents/SpecialButtons/Specials.js

```diff
@@ -1,17 +1,35 @@
-import React from "react";
-
+import React, {useState}  from "react";
+import SpecialButton from './SpecialButton'
 //import any components needed
-
+import {specials} from '../../../data';
 //Import your array data to from the provided data file
 
-const Specials = () => {
+const Specials = ({ updateValue, resetValue, backSpace }) => {
   // STEP 2 - add the imported data to state
+ const [specialsState] = useState(specials)
 
+ const clickSpecialButton = button => {
+   button === 'C' ? resetValue() : updateValue (button)
```

Comment:
A total design choice, but I expliciting writing out the resetValue has it's own merits, but I find it easier if the code is shorter and easier to read if it doesn't provide significant logic to it.